docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -41,6 +41,9 @@ Standard MXNet\* symbols:
 | Symbol Name in MXNet\*| Limitations|
 | :----------| :----------|
 | Activation | supported "act_type" = "relu", "sigmoid", "softrelu" or "tanh" |
+| arccosh | No |
```

Comment:
The operations are sorted differently. All starting from the lower case are located below.